### PR TITLE
Add single user

### PR DESCRIPTION
**Pull request by drouba**

### CODE REVIEW

File: public/create_user.php [hunk 1]
```diff
@@ -1,21 +1,40 @@
...
+        $user->password_digest = password_hash($_POST['password'], PASSWORD_DEFAULT);
 
+    } elseif($_POST['role'] == "instructor"){
+        $user = new User();
+        $user->first_name = $_POST['first_name'];
+        $user->last_name = $_POST['last_name'];
+        $user->email = $_POST['email'];
+        $user->set_role("instructor");
+        $user->password_digest = password_hash($_POST['password'], PASSWORD_DEFAULT);
+    }
```
Comment (jamestjw) [inline public/create_user.php:36]: man 😂  so much repetition !!

File: public/create_user.php [hunk 2]
```diff
@@ -35,18 +52,24 @@
...
 <form method="post">
     <label for="first_name">First Name</label>
-    <input type="text" name="first_name" id="first_name">
+    <input type="text" name="first_name" id="first_name" required>
     <label for="last_name">Last Name</label>
-    <input type="text" name="last_name" id="last_name">
-    <label for="password">Password</label>
-    <input type="password" name="password" id="password">
-    <label for="email">Email Address</label>
+    <input type="text" name="last_name" id="last_name" required>
```
Comment (chris7neves) [inline public/create_user.php:57]: can probably put type="email" in the input tag for a small form of validation

File: public/create_user.php [hunk 3]
```diff
@@ -1,21 +1,40 @@
...
+        $user->student_id = $_POST['student_id'];
+        $user->is_password_changed = 0;
+        $user->password_digest = password_hash($_POST['password'], PASSWORD_DEFAULT);
+    
+    } elseif($_POST['role'] == "ta") {
+        $user = new User();
+        $user->first_name = $_POST['first_name'];
+        $user->last_name = $_POST['last_name'];
+        $user->email = $_POST['email'];
+        $user->set_role("ta");
```
Comment (chris7neves) [inline public/create_user.php:26]: shouldnt you also set is_password_changed for ta and instructors as well?

**Review by chris7neves — CHANGES_REQUESTED** [review]
looks good, just some small changes

File: public/create_user.php (hunk 2 above)
Comment (drouba) [reply public/create_user.php:57]: oh good idea

File: public/create_user.php (hunk 3 above)
Comment (drouba) [reply public/create_user.php:26]: oh yeah true

**Review by chris7neves — APPROVED** [review]

**Pull request merged**